fidesctl/src/fidesctl/cli/__init__.py

```diff
@@ -31,26 +31,42 @@
     default="",
     help="Optional configuration file",
 )
+@click.option(
+    "--local",
+    is_flag=True,
+    help="Runs in a local mode that doesn't utilize server calls.",
+)
 @click.pass_context
-def cli(ctx: click.Context, config_path: str) -> None:
+def cli(ctx: click.Context, config_path: str, local: bool) -> None:
     """
-    The parent group for the Fides CLI.
-    Loads the config and passes it within the context.
+    The parent group for the Fidesctl CLI.
     """
+
+    local_commands = [evaluate, parse, view_config]
+    all_commands = [
+        annotate_dataset,
+        apply,
+        delete,
+        generate_dataset,
+        get,
+        init_db,
+        ls,
+        ping,
+        reset_db,
+        webserver,
+    ] + local_commands
```

Comment:
```suggestion
    api_commands = [
        annotate_dataset,
        apply,
        delete,
        generate_dataset,
        get,
        init_db,
        ls,
        ping,
        reset_db,
        webserver,
    ]
```
`all_commands` is fine for right now, but I think having them distinctly separated will be more valuable long-term.

Reply:
it isn't just "api" commands though, they could also be db commands (i.e. the generate-dataset command)

Reply:
we can rename/split up as needed to in the future, but i'd rather make that iteration when we have to/we have a reason to